clients/tango-cli.py

```diff
@@ -453,8 +461,14 @@ def router():
     parser.print_help()
     sys.exit(0)
 
+if args.ssl:
+    _tango_protocol = 'https'
+    if args.port == 3000:
+        args.port = 443
+
```

Comment:
It would be good to print a warning here for the else branch that the SSL option is not being respected because another port is already being specified